index.html

```diff
@@ -13,6 +13,12 @@
 </head>
 
 <body>
+ <section>
+   <header>
```

Comment:
add class header aqui 
```suggestion
   <header class="header">
```

Reply:
feitoo !! mais com a class  assim <header class" cabeçalho ">